### PR TITLE
added countdown.

### PR DESCRIPTION
**Pull request by Anupam-dagar**
This PR is for the issue #223 

[Live Demo URL]https://anupam-dagar.github.io/opencodecollab/ < -- please add your Url over here

Added a countdown for the event.

[Read once](https://github.com/fossiiita/opencodecollab#contributing)

Note - After this PR gets accepted, please add your contribution to scores.yml . Refer to README for more details.

### CODE REVIEW

File: _includes/navbar.html [hunk 1]
```diff
@@ -246,5 +246,23 @@ <h4 class="modal-title">Trailer of OpenCode</h4>
         </div>
     </div>                                                                 
 </div>
-
-  
+<div class="tcontain">
+<div id="clockdiv">
```
Comment (gauthamzz) [inline _includes/navbar.html]: wrong indendation

File: javascripts/countdown.js [hunk 2]
```diff
@@ -0,0 +1,41 @@
+function getTimeRemaining(endtime) {
```
Comment (gauthamzz) [inline javascripts/countdown.js]: add comments on top

File: stylesheets/countdown.css [hunk 3]
```diff
@@ -0,0 +1,37 @@
+.tcontain{
+	margin-top: 4%;
+	padding: 5%;
+	margin-bottom: -8%;
+	text-align: center;
+	background: #00ECB9;
+  font-family: sans-serif;
```
Comment (gauthamzz) [inline stylesheets/countdown.css]: wrong indendation

**Comment by Anupam-dagar** [comment]
required changes done.

**Comment by Anupam-dagar** [comment]
Added comments in navbar.html

**Comment by Anupam-dagar** [comment]
It's done now

**Pull request merged**